### PR TITLE
HOLD FOR RELEASE: SEARCH-1952: Update install page docs

### PR DESCRIPTION
**Pull request by RuslanKostiv1**
## Purpose of this pull request

Note:  This PR will be closed and recreated for Experience League.

Updates documentation according to the changes in `magento-live-search` modules. 

This pull request (PR) contains changes in `magento-live-search` install/update processes.

## Magento release version

Changes are actual for Magento >=2.4.3

## Product editions

Adobe Commerce only 
Magento Live Search Extension

## Affected documentation pages

- https://docs.magento.com/user-guide/live-search/install.html

## Additional information

_OPTIONAL. What other information can you provide?_

### CODE REVIEW

**Review by ccondra — APPROVED** [review]
Looks good!

**Comment by dobooth** [comment]
@RuslanKostiv1 Thanks for contributing. Please sign the Adobe CLA below please.

**Comment by ccondra** [comment]
This is on hold until the Live Search release.

**Comment by ccondra** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@ccondra the branch with the code is successfully imported into the `magento-devdocs/merchdocs` repository. Branch name: `imported-RuslanKostiv1-merchdocs-1714`.

File: src/live-search/install.md
Comment (ccondra) [inline src/live-search/install.md:231]:
```suggestion
1. If your currently installed `magento/live-search` version is `1.3.1` or below and you are upgrading to version `2.0.0` and higher, run the following command before the upgrade:
```

File: src/live-search/install.md
Comment (ccondra) [inline src/live-search/install.md:231]:
```suggestion
1. If your currently installed `magento/live-search` version is `1.3.1` or below and you are upgrading to version `2.0.0` or higher, run the following command before the upgrade:
```

File: src/live-search/install.md
Comment (ccondra) [inline src/live-search/install.md:236]:
```suggestion
   For information about the currently installed `magento/live-search` version, run the following command:
```

File: src/live-search/install.md
Comment (ccondra) [inline src/live-search/install.md:229]:
```suggestion
To update to a major version, such as from 1.0.0 to 2.0.0, edit the project’s root Composer `.json` file as follows:
```

Comment (ccondra) [inline src/live-search/install.md]:
```suggestion
To update to a major version, such as from 1.0.0 to 2.0.0, edit the project’s root Composer `.json` file as follows:
```

**Comment by dshevtsov** [comment]
Converted to draft until ready for merge

**Comment by ccondra** [comment]
This PR is being closed and moved to Experience League.

**Pull request closed without merging**
